e2e-pw/src/oss/poms/grid/index.ts

```diff
@@ -92,4 +97,14 @@ class GridAsserter {
       { timeout: 2000 }
     );
   }
+
+  async waitForGridToLoad() {
+    return this.gridPom.page.waitForSelector("[data-cy=looker]", {
+      timeout: 2000,
+    });
+  }
```

Comment:
there is no assertion being done here, should you move it out of the asserter class?